src/parser.R

```diff
@@ -1,6 +1,6 @@
-library(stringr)
+library(stringr)   # For text manipulation
 library(testthat)  # For unittesting: test_that
-library(here) # For relative working directory. Looks for where there is a file with .Rproj extension and sets that folder as the root. 
+library(here)      # For relative working directory. Looks for where there is a file with .Rproj extension and sets that folder as the root. 
```

Comment:
is this library needed anymore?

Reply:
We don't need it anymore for this project.